include/swift-c/DependencyScan/DependencyScan.h

```diff
@@ -92,6 +104,10 @@ SWIFTSCAN_PUBLIC swiftscan_dependency_set_t *
 swiftscan_dependency_graph_get_dependencies(
     swiftscan_dependency_graph_t result);
 
+SWIFTSCAN_PUBLIC swiftscan_diagnostic_set_t *
```

Comment:
Do you need to call out if the return value needs to be disposed or not? I think the ones inside import_set and dependency_graph should not be disposed as they will be disposed together with the overall struct.

Reply:
Added a comment. Yes, this one is meant to be disposed of with the entire graph result, yes. Thanks.